storageClient/storageClient.go

```diff
@@ -515,6 +515,9 @@ func (c *StorageClient) WriteRecord(ctx context.Context, params Record) (*Record
 func (c *StorageClient) WriteFile(ctx context.Context, params Record) (*PendingFileResponse, error) {
 	var result *PendingFileResponse
 	path := c.Host + storageServiceBasePath + "/files"
+	if params.Metadata.FileMeta.Compression == "" {
```

Comment:
Quick Question:
omitempty for the fileMeta is for the sever correct? so it wouldn't cause issues here

Reply:
it is for the server, but if 'Compression' isn't included in the fileMeta then the request from WriteFile doesn't actually register the record as a file and causes an error. PDS doesn't actually return an error in this case, so i'm setting this to make sure that it has the required info & this problem won't occur.

Reply:
a comment here would be useful, but we'll want this default safe logic only in the sdk/e3db-go, e3db-clients-go needs to be dumb/ have no implicit behavior

Reply:
@efabens @carrala does Storage service return an error if compression isn't included? If not can you add a ticket for that so that whenever we get to enforcing files to go through storage v2 it will be safer for everyone.

Reply:
That is a great idea. What is really odd is that pds doesn't check it at creation or verification time either. Just at fetch time. @carrala in the storage service endpoints we should validate during the initial post that file meta has all three fields present.